regen-network/regen-web
# chore: implement changes to project create flow buttons

## Description

Closes: regen-network/rnd-dev-team#1871



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2323--regen-marketplace.netlify.app/
1. Login
1. Create or edit a draft project and check the footer has the changes specified in https://github.com/regen-network/rnd-dev-team/issues/1871. In particular, test "Save draft & exit" button
2. Continue until the Review step where the button is "Submit" instead of "Save and next"

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
